backend/inspirehep/records/views.py

```diff
@@ -239,6 +247,37 @@ def reference_self_curation(args):
     return jsonify({"message": "Success"}), 200
 
 
+@blueprint.route("/literature/<path:recid>/<int:old_revision>..<int:new_revision>")
```

Comment:
for ensuring it's a valid record we can use `<inspirepid(lit,record_class="inspirehep.records.api:LiteratureRecord"):pid_value>` this will ensure that we don't pass random values to `get_uuid_for_recid` which might cause vulnerability issues.